.dependabot/config.yml

```
@@ -0,0 +1,18 @@
+version: 1
+update_configs:
+  - package_manager: 'python'
+    directory: '/'
+    update_schedule: 'weekly'
+    ignored_updates:
+      - match:
+          # Some pytest plugins not working
+          dependency_name: 'pytest'
+          version_requirement: '4.x'
+      - match:
+          # Waiting for Bryan to review newest version
+          dependency_name: 'robotframework-seleniumlibrary'
+          version_requirement: '3.x'
+      - match:
+          # No published source tarbal for 0.74.3
+          dependency_name: 'simple-salesforce'
+          version_requirement: '0.74.3'
```

```suggestion
          version_requirement: '0.74.2'
```

0.74.3 is the one we're trying to avoid